### DISCUSSION TITLE
How to get the latest Android version?

### DISCUSSION THREAD

**Discussion by renyuneyun** [mrzmyr/pixy-mood-tracker-app#201] [Q&A] [answered]
Hi,
I accidentally found this app and really loves it. After doing some investigation, I found this repo, and realized it is actually open source (horray!). Thanks for making it!
However, the biggest blockage for me is: how to obtain the latest version (for Android)?
The reason I'm asking is because your most recent [changelog post](https://pixy.hellonext.co/changelog/important-backup-you-data-until-01-06-23) says something may happen for the 1 June release, so I'd like to stick on the safe side.

What I tried:
- Google Play: updated on last December; quite outdated (or not? the Release of v1.61.1 in the repo was in this year?? a mismatch...).
- GitHub: nothing in Release; nothing in Actions.
- Following README:
    - Expo Go does not run correctly on my device;
    - `yarn build:android:dev` requires EAS username, which I have no clue why.
- F-Droid: nothing found; not in that repo.

**Comment by mrzmyr** [comment, marked as answer]
Hey @renyuneyun, I totally understand your questions here: "Open source – but I can't build it?" basically.

- I didn't automate the releases yet, therefore they are not uploaded to F-Droid or GitHub. 
- Also the releases are managed through [expo](https://expo.dev/) which makes it possible to bypass the App Store review (see [documentation](https://docs.expo.dev/eas-update/how-it-works/#publishing-an-update)). And this is the reason why the build version numbers mismatch with the ones from the App Store or Google Play Store. There is an "online" version that is a head of the last native build.
- You should be able to create you're own build by signing up with expo (which is free) and to run the build locally by using `yarn build:android:dev`

I will write up an announcement about the status of the app (and why I've written the changelog post) and did't update yet soon.

Does that answer your questions?

**Reply by renyuneyun** [reply]
Thanks for the explanation. I seem to have updated the App to v1.81.1, by a random method. Probably that's because I scanned the QR code after `yarn start`, and restarted the App? (The scanning opened Pixy; Expo Go App never worked for me.)
Is this the expected way to update it?

Anyway, if that's how it works, I'm fine with it (for now). If Google Play Store reviewing is an issue, maybe it's worth publishing it to some alternative stores, e.g. F-Droid which is for FLOSS Apps.

By the way, is there a way to contribute translation? I found many incorrect ones for Chinese (I'm Chinese native), so would like to improve it incrementally. But collecting them together with a PR seems to be an overkill...
